src/scenes/TaskDetailsView/components/EditTaskPrompt/index.tsx

```diff
@@ -0,0 +1,72 @@
+import {Button, Label, Modal, Textarea, TextInput} from "flowbite-react";
+
+interface EditTaskPromptProps {
+  show: boolean,
+  onClose: () => void,
+}
+
+function EditTaskPrompt({ show, onClose }: EditTaskPromptProps) {
```

Comment:
Let's take the task as a property and populate the form with those data

Reply:
This is a good working solution, well done.
We can further improve performance if we can somehow change this implementation from making another network call to getting the task passed down to the component via `props`. Let's edit the edit task prompt props to include the task in it. And then take that input parameter and render the form. 
Then you'll have to pass down the task object to this component from task details view.